webapp/tests/forms/test_session_data.py

```diff
@@ -40,15 +25,15 @@ def test_if_session_data_and_default_data_different_then_update_session_data(sel
         session_data = get_session_data('form_data', default_data=default_data)
         assert expected_data == session_data
 
-    def test_if_session_data_for_the_current_user_is_returned(self, testing_database):
+    def test_if_session_data_for_the_current_user_is_returned(self, testing_current_user):
```

Comment:
I think we should keep the test for an incorrect identifier, because it is different from the current_user test. wdyt?

Reply:
I will readd it!

Reply:
Thanks